pkgs/by-name/he/heimdall-udev-rules/package.nix

```diff
@@ -0,0 +1,24 @@
+{
+  heimdall,
+  lib,
+  runCommandLocal,
+}:
+
+runCommandLocal "heimdall-udev-rules"
+  {
+    inherit (heimdall) version src;
+
+    meta =
+      (builtins.removeAttrs heimdall.meta [
+        "broken"
+        "mainProgram"
+      ])
+      // {
+        description = "udev rules for download mode on Samsung mobile devices";
+        platforms = lib.platforms.linux;
+      };
```

Comment:
Is there any preference on whether udev rules should be packaged as a completely separate package or whether the package should just have an `udev` output? Making a search within nixpkgs seems to show a wild mix of `.udev` outputs, separate `-udev-rules` packages, and just leaving them in the full package.

Reply:
My guess is that the `-udev-rules` approach predates multiple outputs and people then picked *something* as a base for new packages. My personal preference (not that it matters in this context) is definitely the `.udev` output.

Reply:
Added udev output